resources/campaigns/responses/all_campaigns.yml

```diff
@@ -0,0 +1,46 @@
+description: >-
+  A dictionary with a data property that contains an array of
+  up to `limit` campaigns. Each entry in the array is a separate
+  campaign. The previous and next page of campaigns can be retrieved
+  by calling the endpoint contained in the `previous_url` and
+  `next_url` fields in the API response respectively.<br>If no
+  more campaigns are available beyond the current set of returned results,
+  the `next_url` field will be empty.
+
+content:
+  application/json:
+    schema:
+      allOf:
+        - $ref: "../../../shared/models/list.yml"
+        - type: object
+          properties:
+            data:
+              type: array
+              description: list of campaigns
+              items:
+                $ref: "../models/campaign.yml"
+
+    example:
+      data:
```

Comment:
Redocly: `example value must conform to the schema "data" property type must be an array`